### ISSUE TITLE
Renommage d'un item

### ISSUE THREAD

**Issue by mvdandrieux** [cafecentral/grandcentral#115]
Quant on renomme un item (sa clef), tout suit, sauf la table.

**Comment by mvdandrieux** [comment]
@scux Nous avions décidé que ce ne serait pas possible. Il faut donc prévoir un truc qui empêche l'utilisateur de le faire. On passe le champ key de item en readonly mais une fois le form rempli ou quelque chose comme ça ?

**Comment by scux** [comment]
Oui, il faut faire en sorte qu'on ne puisse pas renommer la clef d'un item.

**Comment by mvdandrieux** [comment]
Une idée server-side ? un truc qui modifie le form à la volée que si item = item et id != null ? sinon j'y vais en js...

**Comment by scux** [comment]
Serveur side, il faut préciser que le champ est readonly.
A mon avis, il faut faire ça du côté des attributs.
